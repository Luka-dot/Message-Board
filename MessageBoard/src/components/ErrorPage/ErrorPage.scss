.Error-Message {
  width: 100%;
  height: 100%;
  top: 0px;
  left: 0px;
  background-color: #e5e5e591;
  position: fixed;
  z-index: 4000;
  transition: opacity 0.218s ease-in 0s;

  &::before {
    content: "";
  }
}

.Error-Content {
  position:fixed;
  background: #ffe680;
  width: 100%;
  max-width: 350px;
  border-radius: 15px;
  height: auto;
  padding: 3rem;
  top: 50%;
  left: 50%;
  transform: translate(-50%,-50%);
  box-shadow: 0px 2px 10px 3px #0000006b;

  h1 {
    margin: 0;
    color: $textColor;
    font-weight: 400;
    padding-bottom: 1em;
    font-size: 24px;
    font-family: $font-Lato;
  }

  a {
    font-family: $font-Abel;
    color: #fff;
    font-size: 1em;
    font-weight: 500;
    text-decoration: none;
    background: #0070B7;
    padding: .2em .7em;
    border-radius: 5px;
    transition: all .2s ease;
    box-shadow: 0px 0px 5px 0px #00000068;
    &:hover {
      background: hsla(0, 0%, 50%, 0.35);
      color: white;
    }
  }

}
