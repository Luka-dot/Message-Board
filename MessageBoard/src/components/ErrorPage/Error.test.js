import React from 'react';
import { ErrorPage } from './ErrorPage';
import { shallow } from 'enzyme';

describe('Error', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ErrorPage />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
