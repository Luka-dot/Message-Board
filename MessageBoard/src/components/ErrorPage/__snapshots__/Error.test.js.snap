// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match the snapshot 1`] = `
<div
  className="Error-Message"
>
  <section
    className="Error-Content"
  >
    <h1>
      Page Not Found
    </h1>
    <NavLink
      to="/"
    >
      Return to Homepage
    </NavLink>
  </section>
</div>
`;
